paddle/fluid/inference/tests/api/int8_mkldnn_quantization.md

```diff
@@ -0,0 +1,88 @@
+# INT8 MKLDNN quantization 
+
+This document describes how to use Paddle inference to convert the FP32 model to INT8 model on ResNet-50 and MobileNet-V1. We provide the instructions on enabling INT8 MKLDNN quantization in Paddle inference and show the ResNet-50 and MobileNet-V1 results in accuracy and performance.
+
+## 0. Prerequisite
+You need to build install at least PaddlePaddle-1.4.The test dataset and model will downloaded automatically in the build process.
+
+## 1. Enable INT8 MKLDNN quantization
```

Comment:
You can refer to the unit test in xxx. Basically, there are xxx steps:

Reply:
Done